modular_skyrat/modules/resleeving/code/research/resleeving_research.dm

```diff
@@ -0,0 +1,14 @@
+/datum/design/rsd_interface
+	name = "RSD brain interface"
```

Comment:
name = "RSD Phylactery"
desc = "A brain interface that allows for transfer of Resonance from a handheld RSD, such as the Evoker model."